core/templates/dev/head/exploration_editor/main_tab/GadgetEditor.js

```diff
@@ -35,6 +35,8 @@ oppia.controller('GadgetEditor', [
       $scope.activeStateName = currentStateName;
     });
 
+    $scope.extensionResourcePrefix = GLOBALS.ASSET_DIR_PREFIX;
```

Comment:
Can you add a method in the url construction service instead, like you did for interactions?